### ISSUE TITLE
Help needed with moving to alpine

### ISSUE THREAD

**Issue by brainless** [appwrite/docker-resque-ui#4]
Hello, This is not a PR for the actual work. But I got stuck somewhere and need a little help. So I tested a Dockerfile with Ruby 2.4 or 2.5 on alpine (standard on Dockerhub) and `resque-web` command runs fine. But when I run the `/entrypoint.sh` it does not work. I get the following error and after than `resque-web` also does not work.

Is there something specific in the entrypoint that is causing this issue?

```
bash-5.0# resque-web
[2020-09-20 09:54:29 +0000] Starting 'resque-web'...
[2020-09-20 09:54:29 +0000] trying port 5678...


bash-5.0# /entrypoint.sh
[2020-09-20 09:54:39 +0000] Using Redis connection 'redis://127.0.0.1:6379'
[2020-09-20 09:54:39 +0000] Starting 'resque-web'...
Traceback (most recent call last):
	15: from /usr/local/bundle/bin/resque-web:23:in `<main>'
	14: from /usr/local/bundle/bin/resque-web:23:in `load'
	13: from /usr/local/bundle/gems/resque-2.0.0/bin/resque-web:13:in `<top (required)>'
	12: from /usr/local/bundle/gems/resque-2.0.0/bin/resque-web:13:in `new'
	11: from /usr/local/bundle/gems/vegas-0.1.11/lib/vegas/runner.rb:77:in `initialize'
	10: from /usr/local/bundle/gems/vegas-0.1.11/lib/vegas/runner.rb:108:in `start'
	 9: from /usr/local/bundle/gems/vegas-0.1.11/lib/vegas/runner.rb:160:in `check_for_running'
	 8: from /usr/local/bundle/gems/vegas-0.1.11/lib/vegas/runner.rb:146:in `port_open?'
	 7: from /usr/local/lib/ruby/2.5.0/open-uri.rb:35:in `open'
	 6: from /usr/local/lib/ruby/2.5.0/open-uri.rb:735:in `open'
	 5: from /usr/local/lib/ruby/2.5.0/open-uri.rb:165:in `open_uri'
	 4: from /usr/local/lib/ruby/2.5.0/open-uri.rb:224:in `open_loop'
	 3: from /usr/local/lib/ruby/2.5.0/open-uri.rb:224:in `catch'
	 2: from /usr/local/lib/ruby/2.5.0/open-uri.rb:226:in `block in open_loop'
	 1: from /usr/local/lib/ruby/2.5.0/open-uri.rb:755:in `buffer_open'
/usr/local/lib/ruby/2.5.0/open-uri.rb:377:in `open_http': 500 Internal Server Error  (OpenURI::HTTPError)
```

**Comment by eldadfux** [comment]
From the top of my head I can think of 2 ideas:
1. Maybe a problem connecting to a Redis instance?
2. Maybe some differences between Ubuntu/Debian bash to alpine sh?

**Comment by brainless** [comment]
@eldadfux Thanks for the ideas.

1. I did not know the Redis server needed to be running since the original Docker container runs by itself without any Redis container
2. I tried with debian (buster) too, I can not remember the exact error, but I think I got the same error. I will double check

Please let me know if you think of something else, I will clarify these soon.

**Comment by eldadfux** [comment]
Solved in #5
